src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -105,6 +106,8 @@ public string QuickPulseAuthenticationApiKey
         /// </summary>
         public bool EnableDependencyTracking { get; set; } = true;
 
+        public DependencyTrackingOptions DependencyTrackingOptions { get; set; }
```

Comment:
Add a summary comment here. And be sure to mention that this doesn't apply if `EnableDependencyTracking` is `false`.

Reply:
Fixed